src/OrchardCore/OrchardCore.HealthChecks.Abstractions/HealthChecksOptions.cs

```diff
@@ -1,18 +1,17 @@
-namespace OrchardCore.HealthChecks
+namespace OrchardCore.HealthChecks;
+
+/// <summary>
+/// Represents a programmable health chekcs options.
+/// </summary>
+public class HealthChecksOptions
 {
     /// <summary>
-    /// Represents a programmable health chekcs options.
+    /// Gets or sets the health check URL. Default to "/health/live".
     /// </summary>
-    public class HealthChecksOptions
-    {
-        /// <summary>
-        /// Gets or sets the health check URL. Default to "/health/live".
-        /// </summary>
-        public string Url { get; set; } = "/health/live";
+    public string Url { get; set; } = "/health/live";
 
-        /// <summary>
-Gets or sets a value indicating whether to show the details for the checks dependency or not. Defaults to <see langword="false"/>.
-        /// </summary>
-        public bool ShowDetails { get; set; }
-    }
+    /// <summary>
+    /// Gets or sets a value indicating whether to show the detailed information (name, status and description) for the checks dependency or not. Defaults to <see langword="false"/>.
+    /// </summary>
+    public bool ShowDetails { get; set; }
```

Comment:
So didn't [we discuss](https://github.com/OrchardCMS/OrchardCore/pull/13590#discussion_r1565006360) that having a health check response without details doesn't make sense? Because then we don't need this config, and the feature should work as if it were permanently `true`.

Reply:
By default, as we are doing today, hitting the health check the app status, the newly added flag is used if you want to explore all the health checks in the system with their statuses

Reply:
I understand what this does, but not why we'd need it (that it didn't do this before is not a reason in itself).

Reply:
We didn't have any health checks before, but now we do, that's useful to show all the health check details either as JSON or probably later from UI

Reply:
Yes, I'm clear on that. Why do we need the `ShowDetails` option though, when we previously agreed that it doesn't make sense to have it `false`?

Reply:
Got it that why you pushing to use 200 :)